x-pack/platform/plugins/shared/lock-manager/kibana.jsonc

```diff
@@ -0,0 +1,6 @@
+{
+  "type": "shared-common",
+  "id": "@kbn/lock-manager",
+  "owner": ["@elastic/obs-ai-assistant"],
+  "devOnly": true
+}
```

Comment:
This manifest looks like a package. Not a plugin.
Plugins have the `"plugin"` object, and `type: "plugin"`.